www/docs/getting_started.md

```diff
@@ -2,6 +2,151 @@
 
 ## Installation
 
-## Explore a finished service
+Before you begin, we recommend installing the NPM packages for the Skip API, server, client, and helpers.
 
-## Write a reactive function
+```npm install @skipruntime/api @skipruntime/server @skipruntime/client @skipruntime/helpers```
+
+## Introduction
+
+We have several resources available to help you learn the Skip framework, depending on your bcakground and preferences.
+
+This guide takes a "top-down" approach, showing you how to use our latest APIs in an idiomatic and practical way.  We recommend that most users start here.
```

Comment:
> *latest* APIs

This seems to hint that those will change often. While it may be true in the short term, I'm not sure we should set it as a general expectation.